bats/tests/containers/wasm.bats

```diff
@@ -114,7 +114,9 @@ download_shim() {
     local filename="containerd-wasm-shims-${version}-${shim}-linux-${ARCH}.tar.gz"
 
     mkdir -p "$PATH_CONTAINERD_SHIMS"
-    curl --silent --location --output "${PATH_CONTAINERD_SHIMS}/${filename}" "${base_url}/${filename}"
+    # On Windows, use curl from the WSL distro instead of curl.exe to ensure we
+    # have no issues with PATH_CONTAINERD_SHIMS being in Linux format.
+    command curl --location --output "${PATH_CONTAINERD_SHIMS}/${filename}" "${base_url}/${filename}"
```

Comment:
I would prefer to convert the path to host format instead:

```suggestion
    curl --location --output "$(host_path "${PATH_CONTAINERD_SHIMS}/${filename}")" "${base_url}/${filename}"
```

It seems more consistent to use the `curl.exe` all the time on Windows.